root/etc/e-smith/templates/etc/ssh/sshd_config/70Restricted2Sftp

```diff
@@ -0,0 +1,24 @@
+{
+    # restrict groups to sftp, ssh access is refused
+    use strict;
+    use esmith::ConfigDB;
+    my $db = esmith::ConfigDB->open_ro();
+
+    my @SftpRestricted = ();
+    foreach ( split(',',$db->get_prop('sshd','AllowGroups') || '')) {
+        my ($group, $sftp) = split(':', $_);
+        if(!$group || !$sftp) {
+            next;
+        }
+        push @SftpRestricted, $group if ($sftp eq 'sftp');
+    }
+
+    if (@SftpRestricted) {
+        my $matchGroup = join (",", map { $_ } @SftpRestricted);
```

Comment:
Protect spaces and add `@domain` suffix here too?

Reply:
IIRC to protect spaces (I will test it again) is "domain admins@domain.org,famille@domain.org,etc@domain.org"

Reply:
done